src/app/app.component.html

```diff
@@ -15,6 +15,7 @@ <h1>{{title}}</h1>
 </main>
 
 <footer class="typography">
-  <p>NuschtOS Search</p>
+  <p>NüschtOS Search</p>
+  <p>Licensed under MIT</p>
```

Comment:
I would like to put this behind the source link and in brackets.

Reply:
You mean I should switch this line with the next and WDYM with brackets?